docs/core-manage-plugins.md

```diff
@@ -1,24 +1,25 @@
-Plugins are how asdf understands handling different packages.
+Plugins are how asdf knows how to handle different packages.
 
-See [All plugins](plugins-all) for the plugins repository which lists all asdf plugins we know.
-
-See [Creating Plugins](plugins-create) for the super-simple API for supporting more languages.
+See [Creating Plugins](plugins-create) for the plugin API used to support more languages.
```

Comment:
Above we say "packages" and here we use the word "languages". Not some of the things asdf versions are not languages. "support more tools" or "support more tools and languages" might be better.